src/App.js

```diff
@@ -0,0 +1,47 @@
+import {
+  BrowserRouter as Router, Routes, Route, Outlet,
+} from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './Redux/store';
+import './App.css';
+import SideBar from './components/sidebar/SideBar';
+import HousesDetails from './components/houses details/HouseDetails';
+import Houses from './components/houses/Houses';
+import Login from './components/auth/Login';
+import Register from './components/auth/Register';
+import AddHouse from './components/add hosue/addHouse';
+import DeleteHouse from './components/delete house/deleteHouse';
+import Reservation from './components/reservation/Reservation';
+import MyReservations from './components/my reservations/MyReservations';
+
+const SidebarLayout = () => (
+  <>
+    <SideBar />
+    <Outlet />
+  </>
+);
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/Wahaj-Ali/homely-frontend/assets/57421823/75f07bd4-f666-4983-8dc1-9ddbe620344f)

Reply:
Thanks @Meltrust for your valuable suggestions. I will try my best to implement the changes.